### ISSUE TITLE
Fix feature outline on hover

### ISSUE THREAD

**Issue by AliyanH** [Maps4HTML/Web-Map-Custom-Element#919]
When you hover over the features on the map given below, the outline in some places does not show properly due to overlap, we can get around this with messing with Z-index or dom order for the outline as required to show the outline at the top of DOM order at all times.

https://maps4html.org/experiments/api/geojson/

**Comment by AliyanH** [comment]
related to #940.

It appears SVG doesn't directly support z-index like CSS does ([link](https://gsap.com/community/forums/topic/22566-z-index-in-svg-element/#:~:text=Z%20index%20cannot%20work%20with,ones%20being%20put%20on%20top.)). I played around by moving the hovered element to the end of its parent container. Since SVGs render the last element in the markup on top, and this seems to solve the issue, but may present another issue when features overlap. For discussion.
